com.chartboost.mediation/Runtime/Banner/ChartboostMediationBannerAndroid.cs

```diff
@@ -18,8 +19,23 @@ public sealed class ChartboostMediationBannerAndroid : ChartboostMediationBanner
         public ChartboostMediationBannerAndroid(string placementName, ChartboostMediationBannerAdSize size) : base(placementName, size)
         {
             LogTag = "ChartboostMediationBanner (Android)";
+            
+            if (size.Name == "ADAPTIVE")
+            {
+                Logger.LogError(LogTag,$"Adaptive sizes are not supported for `ChartboostMediationBannerAd`. Use `ChartboostMediationBannerView` instead");
+                return;
+            }
+            
+            var fixedSize = size.Name switch
+            {
+                "STANDARD" => 0,
+                "MEDIUM" => 1,
+                "LEADERBOARD" => 2,
```

Comment:
We could convert this values to an enum:

```csharp
    public enum ChartboostMediationBannerName
    {
        Adaptive = -1,
        Standard = 0,
        Medium = 1,
        Leaderboard = 2,
    }
```
As it would be more efficient to switch through the toggles than comparing strings. It would also make it easier to know where such implementations happen and we would be less likely to make typos/mistakes.